packages/@ourworldindata/grapher/src/mapCharts/MapChart.tsx

```diff
@@ -295,6 +295,13 @@ export class MapChart
         if (!ev.shiftKey) {
             this.selectionArray.setSelectedEntities([entityName])
             this.manager.currentTab = GrapherTabOption.chart
+            if (
+                this.manager.isLineChartThatTurnedIntoDiscreteBar &&
+                this.manager.hasTimeline &&
+                this.manager.resetHandleTimeBounds
+            ) {
+                this.manager.resetHandleTimeBounds()
```

Comment:
```suggestion
            if (
                this.manager.isLineChartThatTurnedIntoDiscreteBar &&
                this.manager.hasTimeline
            ) {
                this.manager.resetHandleTimeBounds?.()
```

I _slightly_ prefer this, but both are fine really.